shell/app-shell/elements/remote-friends-shared-handles.js

```diff
@@ -91,35 +91,46 @@ class RemoteFriendsSharedHandles extends Xen.Base {
     const tagString = (tags && tags.length ? `${tags.sort().join('_').replace(/#/g, '')}` : '');
     // only box if we have values and tags
     if (values && tagString) {
+      // acquire type record
+      const arcsType = ArcsUtils.typeFromMetaType(type);
```

Comment:
I wonder if this should be `ArcsUtils.typeFromMetaType(type.resolvedType()`? I think that handles the case where it's a type reference instead of a concrete type. @shans what do you think?

I ran into that elsewhere and put a review up as #898.

Reply:
It's confusing, but the change you made was to invocation of `metaTypeFromType` and this is the converse.

Reply:
Haha. Missed that!